NEWS.md

```diff
@@ -27,9 +32,6 @@
 * Allow selection of all character and factor variables in the dataset, except for the variables ending with '_id', to use for filters and stratifications.
 * Adding a function `pad_signals()` performing the padding of signals timeseries (extension of threshold and expecation to the past for visualisation) which was previously done in the signals tab server.
 * adding method and number_of_weeks to the resulting tibble of `get_signals()` to reduce number of parameters needed to be passed forward.
-* Improved wording of objects in signals and reports tab
```

Comment:
Probably just a volatility error: can you move the deleted points back to 0.2.0 and just add your new points to 0.2.1? Thanks

Reply:
Sure!